package-lock.json

```diff
@@ -5524,6 +5524,12 @@
       "integrity": "sha512-dRLjCWHYg4oaA77cxO64oO+7JwCwnIzkZPdrrC71jQmQtlhM556pwKo5bUzqvZndkVbeFLIIi+9TC40JNF5hNQ==",
       "dev": true
     },
+    "node_modules/@types/lodash": {
```

Comment:
Why are you adding this to the package lock?

Reply:
Well, I believe this was introduced before our talk. Also, it was in the PR (1). I will revert it in this PR.